corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -154,10 +154,32 @@ FormplayerFrontend.on('startForm', function (data) {
     };
     data.onsubmit = function (resp) {
         if (resp.status === "success") {
-            var markdowner = window.markdownit();
+            var markdowner = window.markdownit(),
+                reverse = hqImport("hqwebapp/js/initial_page_data").reverse,
+                analyticsLinks = [
+                    { url: reverse('list_case_exports'), text: '[Data Feedback Loop Test] Clicked on Export Cases Link' },
+                    { url: reverse('list_form_exports'), text: '[Data Feedback Loop Test] Clicked on Export Forms Link' },
+                    { url: reverse('case_details', '.*'), text: '[Data Feedback Loop Test] Clicked on Case Data Link' },
+                    { url: reverse('render_form_data', '.*'), text: '[Data Feedback Loop Test] Clicked on Form Data Link' },
+                ],
                 inTest = hqImport('analytix/js/kissmetrix').getAbTest('Data Feedback Loop') === 'data_feedback_loop_on',
-                message = inTest ? resp.submitResponseMessage : gettext("Form successfully saved!");
+                message = (inTest ? resp.submitResponseMessage : gettext("Form successfully saved!")),
+                dataFeedbackLoopAnalytics = function(e) {
+                    var $target = $(e.target);
+                    if ($target.is("a")) {
+                        var reverse = hqImport("hqwebapp/js/initial_page_data").reverse,
+                            href = $target.attr("href");
+                        _.each(analyticsLinks, function(link) {
+                            if (href.match(RegExp(link.url))) {
+                                $target.attr("target", "_blank");
+                                hqImport('analytix/js/kissmetrix').track.event(link.text);
+                            }
+                        });
+                    }
+                };
```

Comment:
(same question about restricting this per-environment)

Reply:
This is okay - the `getAbTest` call higher up will return `undefined` for servers that don't use analytics.